ansible/mnesia_snapshot.yml

```diff
@@ -9,6 +9,8 @@
     snapshot_suffix: snapshot
     snapshot_filename: "mnesia_{{ env }}_v-{{ db_version }}_{{ ansible_ssh_host|default(inventory_hostname) }}_{{ snapshot_suffix }}.tgz"
     snapshot_path: "{{ snapshots_dir|default('/tmp/snapshots') }}/{{ snapshot_filename }}"
+    snapshot_hash_filename: "{{ snapshot_filename }}.md5"
+    snapshot_hash_path: "{{ snapshots_dir|default('/tmp/snapshots') }}/{{ snapshot_hash_filename }}"
```

Comment:
IMHO this should not include the filename

Reply:
df85214